recipes/ogdf/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "ogdf" %}
+{% set version = "2022.02" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ogdf/ogdf/archive/refs/tags/dogwood-202202.tar.gz
+  sha256: 308cc2749c6a63520f7979bac86a04066dfea2fb9d3a5e89831318db404bfbf5
+
+  patches:
+    - cmake.patch
```

Comment:
Maybe it would make the patch file easier to maintain (GitHub doesn't even want to display it online due to its size) if you only put actual changes to files' contents into the diff and then delete the respective folders in the script?

Reply:
I agree. Please only include the changes to the CMake files in this diff, and do the file deletions in the build script. Otherwise, the patch seems to be 99% file deletion.